### DISCUSSION TITLE
Emanuele Stoppa - Biome

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#210] [Announcements]
- https://twitter.com/ematipico
- https://www.linkedin.com/in/emanuelestoppa
- https://github.com/ematipico
- https://biomejs.dev/

## Topics

- Biome
- Astro

## Questions

- [x] How did you get involved with Rome?
- [x] What was rome and what did it intend to do? 
- [x] What happened to the company?
- [x] Why did you start Biome? 
- [x] Who are you working for now?
- [x] What types of things are unlocked by creating a set of tools that all work off the AST?
- [x] How "production ready" are the linter and the formatter?
- [x] Can I write plugins?
- [x] What does the analyzer do?
- [x] What other tools do you plan to build into biome? Is it still the plan that a test runner, bundler, and transformer are still in store?
- [x] What do you think about bun? Does biome plan to only support node or will it work for other runtimes?
- [x] Whats your favorite thing about biome?
- [x] What are some things biome does to keeep things fast? What is the daemon doing?

### Future

- [x] In the next year what the goal for biome? What do you want to build next?
- [ ] What's your spiciest dev take?
- [ ] What do you think the future of JavaScript tooling is?

**Comment by just-be-dev** [comment]
Tooltips:
- https://github.com/orbitinghail/sqlsync
- https://smith.langchain.com/hub

**Comment by hipstersmoothie** [comment]
https://v0.dev/
https://github.com/sxyazi/yazi

**Comment by ematipico** [comment]
Tooltips: 
- https://blog.jetbrains.com/rust/2023/09/13/introducing-rustrover-a-standalone-rust-ide-by-jetbrains/
